src/std/text.ab

```diff
@@ -100,6 +100,62 @@ pub fun contains(text: Text, phrase: Text): Bool {
   return result == "1"
 }
 
+/// Checks if an array value is in the text.
+pub fun contains_any(text: Text, terms: [Text]): Bool {
+    for term in terms {
+        if contains(text, term) {
+            return true
+        }
+    }
+
+    return false
+}
+
+/// Checks if all the arrays values are in the string
+pub fun contains_all(text: Text, terms: [Text]): Bool {
+    for term in terms {
+        if not contains(text, term) {
+            return false
+        }
+    }
+
+    return true
+}
+
+/// Match all occurences of a regex pattern.
+///
+/// Function uses `sed`
+pub fun match_regex(source: Text, search: Text, extended: Bool = false, match: Text = "p"): Bool {
```

Comment:
I'm sorry, but this function still doesn't need a `match` parameter.  You've previously argued that it gives the user more control over the regex operation, but that just isn't true; any other `sed` command than the default `p` will just mess things up.  Please remove.